.github/issue_template.md

```diff
@@ -0,0 +1,39 @@
+#### Things to consider
+
+1. Please check that you are using the latest scapy version (e.g. installed via pip install https://github.com/secdev/scapy)
```

Comment:
Could you link code markdown ? aka \`stuff\`

Reply:
maybe its a little late - but I don't get what you request... :smile: 

a. should I add a link that directs the user to a cheat sheet that explains how to use code format (I guess this would be useful anyway)

b. should format the links? (those templates are usually shown as plain text so the user wont benefit from the additional formatting)

Reply:
Sorry I wasn't clear, It's easy for users to copy pase code formated as
`code`
It would be good if you could format the command pip... as
`pip install https://github.com/secdev/scapy`
so they can copy/paste it easily

Reply:
c2b5e97 moves the cmd to a separate line formatted as code